gruppi/viste.py

```diff
@@ -10,21 +9,92 @@
 from base.utils import poco_fa
 from gruppi.elenchi import ElencoMembriGruppo
 from gruppi.models import Gruppo, Appartenenza
+from gruppi.forms import ModuloGruppoSpecifico, ModuloGruppoNonSpecifico
+from attivita.models import Attivita, Area, Partecipazione, Turno
+from anagrafica.permessi.applicazioni import DELEGATO_AREA
+from anagrafica.permessi.costanti import GESTIONE_ATTIVITA_AREA
+
+
+@pagina_privata(permessi=GESTIONE_ATTIVITA_AREA)
+def attivita_gruppo(request, me):
+    """
+    Crea gruppi di lavoro per il mio comitato o elimina quelli di cui sono responsabile.
+    """
+    # Sono responsabile di queste aree di intervento.
+    area_permessi = me.oggetti_permesso(GESTIONE_ATTIVITA_AREA)
+    id_sede_aree = area_permessi.values_list('sede_id', flat=True)
+
+    # Attività nelle aree di cui sono responsabile.
+    attivita_specifica = Attivita.objects.filter(area__in=area_permessi)
```

Comment:
Vedi il commento sotto - questo puo' essere semplificato, modificandolo in un nuovo permesso.